src/Role_Command.php

```diff
@@ -253,26 +252,28 @@ public function reset( $args, $assoc_args ) {
 
 		self::persistence_check();
 
-		if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'all' ) && empty( $args ) )
-			WP_CLI::error( "Role key not provided, or is invalid." );
+		if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'all' ) && empty( $args ) ) {
```

Comment:
Import `WP_CLI\Utils` at the top.
```suggestion
		if ( ! Utils\get_flag_value( $assoc_args, 'all' ) && empty( $args ) ) {
```